commons-test/src/main/java/org/infinispan/commons/test/TestSuiteProgress.java

```diff
@@ -96,7 +96,7 @@ synchronized void progress(String color, CharSequence message, Throwable t) {
          actualColor = color;
          actualReset = RESET;
       }
-      out.printf("%s[OK: %5s, KO: %5s, SKIP: %5s] %s%s%n", actualColor, succeeded.get(), failed.get(), skipped.get(), message, actualReset);
+      out.printf("%s[OK:%5s, KO:%3s, SKIP:%3s] %s%s%n", actualColor, succeeded.get(), failed.get(), skipped.get(), message, actualReset);
```

Comment:
I guess if we have more than 999 failures, who cares about alignment :)

Reply:
I've got 999 problems, but alignment ain't one!

Reply:
I hope it doesn't mess with anyone's scripts, but I have one that adds an escape sequence to go up one line after a test progress line, so the more progress messages fit into a terminal line, the less scrolling.